sdk/samples/core/mosquitto_rpc_server_sample.c

```diff
@@ -184,11 +131,12 @@ az_mqtt5_rpc_status execute_command(unlock_request req)
 
 /**
  * @brief Check if there is a pending command and execute it. On completion, if the command hasn't
- * timed out, send the result back to the hfsm
- * @note Result to be sent back to the hfsm needs to be modified for your solution
+ * timed out, send the result back to the HFSM
```

Comment:
We should be consistent with naming and avoid using HFSM which is an implementation detail.
Maybe "send results back to the mqtt_callback?" or rpc_server callback?